demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
```

Comment:
Actually, it appears that this file is not meant to be executed so I think the shebang should be removed (and if the executable bit is set, that should be removed too).